baselines/webworker.generated.d.ts

```diff
@@ -450,6 +450,15 @@ declare var FileReader: {
     new(): FileReader;
 }
 
+interface FormData {
+    append(name: string, value: string | Blob, fileName?: string): void;
+}
+
+declare var FormData: {
+    prototype: FormData;
+    new (form?: HTMLFormElement): FormData;
```

Comment:
@mhegazy [The IDL spec](https://xhr.spec.whatwg.org/#interface-formdata) says `Constructor(optional HTMLFormElement form)`. So I think it should be empty rather than a `string`.